### ISSUE TITLE
Is it possible to use CSS Transitions for node animations?

### ISSUE THREAD

**Issue by kitao** [kitao/divsugar#15]
Need to check whether it is possible to use CSS Transitions instead of JavaScript animations.

**Comment by boblemarin** [comment]
If I understood correctly what divsugar is about, CSS Transitions should be fine. You even get AnimationEvents when they are finished.

**Comment by kitao** [comment]
Thank you for your comments. I'll think about the design of new animation method with CSS Transition using examples from other project. I may also need to consider using CSS Animation.

**Comment by boblemarin** [comment]
If you start working with CSS Animations and Transitions, you might be interested (or inspired) by this little repo : https://github.com/boblemarin/AnimationEvents that helps working with browser prefixed events.

**Comment by kitao** [comment]
I've tried the both of CSS Transitions and CSS Animations. Actually the result is not good...

First, CSS Transitions are difficult to play as a sequence of animations. I tried to use its event handler to detect its end timing, but animations pauses for a few moments before starting next animation.

Next, regarding CSS Animations, it is very troublesome to handle animations dynamically. Because it doesn't support the way of specifying positions as relative coordinates. So new keyframes should be defined for each animation. And it also seems that Chrome doen't support 'animation-fill-mode' now. Because of that, when animations finished, elements go back to the start position.

In conclusion, it is difficult to support CSS Transitions and CSS Animations as APIs of DivSugar for now.
(Of course, users can use them via div property anytime like this: someNode.div.style.webkitTransition = '...')
